packages/material-ui-lab/src/Rating/Rating.js

```diff
@@ -71,18 +72,7 @@ export const styles = (theme) => ({
   /* Pseudo-class applied to the root element if keyboard focused. */
   focusVisible: {},
   /* Visually hide an element. */
-  visuallyhidden: {
-    border: 0,
-    clip: 'rect(0 0 0 0)',
-    height: 1,
-    margin: -1,
-    color: '#000',
-    overflow: 'hidden',
-    padding: 0,
-    position: 'absolute',
-    top: 20,
-    width: 1,
-  },
+  visuallyhidden: visuallyHidden,
```

Comment:
Didn't wanted to create a breaking change for renaming this to camel case, so I just left it like this.

Reply:
I think that for consistency, we could. I doubt people will override this style rule anyway.

Reply:
Done